### PR TITLE
ldap_entry bugfix

### PR DESCRIPTION
**Pull request by gustavomcarmo**
##### SUMMARY
Fixes the same issue reported in #39569, but for the ldap_entry module.

##### ISSUE TYPE
 - Bugfix Pull Request

##### COMPONENT NAME
ldap_entry

##### ANSIBLE VERSION
```
2.6
```

### CODE REVIEW

File: lib/ansible/modules/net_tools/ldap/ldap_entry.py [hunk 1]
```diff
@@ -145,9 +145,9 @@ def _load_attrs(self):
                 attrs[name] = []
 
             if isinstance(value, list):
-                attrs[name] = value
+                attrs[name] = [to_bytes(v) for v in value]
```
Comment (jtyr) [inline lib/ansible/modules/net_tools/ldap/ldap_entry.py]: Please could you use `map(to_bytes, value)` instead of the `for` loop?

**Review by jtyr — CHANGES_REQUESTED** [review]

File: lib/ansible/modules/net_tools/ldap/ldap_entry.py (hunk 1 above)
Comment (gustavomcarmo) [reply lib/ansible/modules/net_tools/ldap/ldap_entry.py]: Done

**Review by jtyr — APPROVED** [review]

**Comment by jtyr** [comment]
Thanks for your PR, @gustavomcarmo.

@ansibot please `shipit`.

**Comment by gustavomcarmo** [comment]
@Akasurde could you please review this PR and merge it?

**Comment by Akasurde** [comment]
Closing and re-opening for CI trigger.

**Comment by Saloshp** [comment]
The fix does not resolve the bug, see updated traceback #45417

**Pull request merged**